headers/types/dungeon_mode/dungeon.h

```diff
@@ -1753,6 +1759,8 @@ struct dungeon {
     undefined field_0x2ca27;
     undefined field_0x2ca28;
     undefined field_0x2ca29;
+    // 0x2CA0C: Probably a char[] that holds the name for the monster that fainted.
+    // Uncertain of exact size.
```

Comment:
If it holds the name of a monster, then it's probably 10 characters long, but if we don't know for sure, I guess it's better to leave it like this.

Reply:
It calls GetMonsterName with this location as the buffer, but it seemed longer than 10 characters and that function still says unverified and imported from Irdkwia's notes. So, I was wary to say definitively. If you would like to see for yourself it's done in 0x022ECDE4

Reply:
I would rather just make this a `char[10]` and specify that the exact size is a guess. It's more descriptive that way.